kong/init.lua

```diff
@@ -1375,6 +1376,26 @@ function Kong.serve_cluster_listener(options)
 end
 
 
+local stream_plugins_api_loaded = false
```

Comment:
Just curious, what's the reason for loading the endpoints lazily. One reason I see loading them during the `init` phase is to allow a bit more RAM to be saved due to the CoW behavior of `fork()`. It may also make testing during development easier to see because Kong will fail to start immediately for obvious things such as syntax error, instead of having to trigger the call later via other means.

Reply:
no real good reason.  on the previous (http-like) design, i got some issues because it seemed that the endpoint table could interfere with the admin api extensions, so i put it on a very different place, and where I was sure it would only happen under the stream module.

this version is much simpler, so it's easier to put in the right place.  fixed